prem_utils/connectors/prem.py

```diff
@@ -14,37 +16,11 @@ class PremConnector(BaseConnector):
     def __init__(
         self,
         api_key: str,
-        base_url: str = "https://ml-development.prem.ninja/api/ml",
+        base_url: str = "https://premai-io--slm-chat-completion-dev.modal.run/",
```

Comment:
Modal should not expose an endpoint. We should pass through ml.premai.io for everything CC @nsosio

Reply:
Yep, I agree with @filopedraz here we need to point to prem-ml endpoint so that we can properly authenticate the requests through prem-ml authentication

Reply:
Sorry guys, this update was not needed. We don't use a web_endpoint in modal anymore. We call the models via function invocation. 

I have reverted the `base_url`. @nsosio @filopedraz

Reply:
Updated @nsosio @filopedraz . Is this ok?

Currently
1. prem-ml connects directly to modal (doesn't go via prem-ml)
2. Prem-utils connects directly to modal 
3. Modal does not have a web-endpoint